cmake/SetupCodeMetrics.cmake

```diff
@@ -49,7 +49,28 @@
 # Note: Only supported for gnu or clang.
 ########################################
 if (ENABLE_COVERAGE)
-    include(blt/cmake/CodeCoverage.cmake)
+    ##########################################################################
+    # Setup coverage compiler flags 
+    ##########################################################################
+    # Set the actual flags for coverage in the COVERAGE_FLAGS variable 
+    # Note: '--coverage' is equivalent to '-fprofile-arcs -ftest-coverage' for compiling and '-lgcov' for linking
+    # Additional flags that might be useful: 
+    #       " -fno-inline -fno-inline-small-functions -fno-default-inline"
+    blt_append_custom_compiler_flag(FLAGS_VAR   COVERAGE_FLAGS 
+                                    DEFAULT     "--coverage")
```

Comment:
The ``blt_append_custom_compiler_flags`` macro allows you to set one flag per supported compiler family.  You should avoid setting a flag for a compiler that is not supported.  E.g.  if ``--coverage`` does not work for intel, it could give a compiler error.  setting ``INTEL " "`` should fix this.